client/imports/login/login.html

```diff
@@ -0,0 +1,10 @@
+<h1> Tuxlab Homepage </h1>
```

Comment:
This page has to standalone of the rest of the template, so it should not be an import.